src/Employer/Employer.Web/Program.cs

```diff
@@ -16,11 +16,7 @@ public static void Main(string[] args)
             try
             {
                 logger.Info("Starting up host");
-                var host = CreateWebHostBuilder(args).Build();
-
-                CheckInfrastructure(host.Services, logger);
-

```

Comment:
The CheckInfrastructure() also creates the tablestorage. Do we not want this?

Reply:
I am not merging this, we don't have tablestorage turned on in AT. I am just trying to find what the issue is. I need a branch to deploy to AT and the best way for it to create the initial release is to create a PR.